admin_guide/managing_networking.adoc

```diff
@@ -460,10 +460,10 @@ endif::openshift-origin[]
   nodeSelector:
     site: springfield-1 <6>
 ----
-<1> The `pod.network.openshift.io/assign-macvlan annotation` creates a Macvlan
-network interface on the primary network interface, and then moves it into the
-pod's network name space before starting the *egress-router* container. Preserve
-the quotation marks around `"true"`. Omitting them results in errors.
+<1> Creates a Macvlan network interface on the primary network interface, and
+moves it into the pod's network name space before starting the *egress-router*
```

Comment:
s/name space/project (Maybe? Also I see namespace as one word elsewhere in the docs.)